integration-tests/README.md

```diff
@@ -0,0 +1,107 @@
+# Integration tests for Oracle WebLogic Server Kubernetes Operator
+This documentation describes the steps to run integration tests for the Oracle WebLogic Server Kubernetes Operator locally on a Oracle Linux environment. The tests are written in the JUnit5 framework and driven by maven profile.
+
+# Directory Configuration and Structure
+Directory structure of integration test source is distributed as follows 
+```
+weblogic-kubernetes-operator/integration-tests - location of module pom.xml  
+weblogic-kubernetes-operator/integration-tests/src/test/java/oracle/weblogic/kubernetes - integration test(JUnit5) classes and utility classes  
+weblogic-kubernetes-operator/integration-tests/src/test/resources - properties, YAML files and other bash scripts
+```
+
+# How does WebLogic Kubernetes Operator integration test work
+ - Build WebLogic Kubernetes Operator image from the downloaded git branch.
+ - Install the latest version of WebLogic Deploy Tooling (WDT) and WebLogic Image Tool (WIT)
+ - Install supported istio service mesh.
+ - Pull the specified version of WebLogic image from container-registry.oracle.com (OCR)
+ - Install WebLogic Kubernetes Operator.
+ - Build a WebLogic domain resource.
+ - Make sure WebLogic domain is running by verifying the corresponding server pod and kubernetes service status. 
+ - After test execution, delete all the Namespaces, Kubernetes Objects created during test execution.
+ - Archive the test standard output for each test class and kubernetes object details for each test method in the diagnostic directory for triage when tests fail or when the env variable COLLECT_LOGS_ON_SUCCESS is set to true.
+ 
+# How to run Operator integration tests locally on Oracle Linux
+
+- Install supported version of Helm and Kubernetes cluster. For detail see [Weblogic Kubernetes Operator guide](https://oracle.github.io/weblogic-kubernetes-operator/userguide/introduction/introduction/).
```

Comment:
For detail see [Weblogic Kubernetes Operator guide] -> For details, see the [WebLogic Server Kubernetes Operator guide]

Reply:
done

Reply:
Not done. For details, see the [WebLogic Server Kubernetes Operator guide]

Reply:
See the link  https://github.com/oracle/weblogic-kubernetes-operator/blob/6719d05560ec120faeaf951ca9354543e6449491/integration-tests/README.md#how-to-run-operator-integration-tests-locally-on-oracle-linux

Reply:
[modified] 
Install supported version of Helm and Kubernetes cluster. For details, see the [WebLogic Server Kubernetes Operator guide](https://oracle.github.io/weblogic-kubernetes-operator/userguide/introduction/introduction/).